app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("cinema")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name}  (first_name, last_name)"
+            f"VALUES (?, ?)",
+            (first_name, last_name)
+
+        )
+        self.connection.commit()
+
+    def all(self) -> list:

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]